src/components/SignInComponent.jsx

```diff
@@ -0,0 +1,16 @@
+import SignIn from '../components/UserSignIn';
+import SignUp from '../components/UserSignUp';
+import GoogleSignIn from '../components/GoogleSignIn';
+
+const SignInComponent = () => {
+  return (
+    <div className="Login">
+      <h1>Log In</h1>
+      <GoogleSignIn></GoogleSignIn>
+      <SignIn></SignIn>
+      <SignUp></SignUp>
+    </div>
+  );
+};
+
+export default SignInComponent;
```

Comment:
this one too. make a 'SignIn' folder under the 'pages' folder and move this component under it